### ISSUE TITLE
[IDEA] Copy result / solution from previous exercise

### ISSUE THREAD

**Issue by Tuxified** [gausby/workshop#13]
In case each workshop exercise builds up on the work of the previous assignment, it might be useful to copy over work from previous exercise. This is just an idea, haven't stumbled upon such a use case yet.

**Comment by gausby** [comment]
I agree, but I have one concern: In a class situation, if we bring the solution from the previous exercise over to the next it would require all the students to finish at the same time. Sometimes if the class has a schedule it may be necessary to skip an exercise—For that reason I've thought about adding a `--force` flag to the next command, which would skip the solution check and just advance to the next exercise.

But, of course for learning at home, on your own time, it could be neat; and I have enjoyed doing exercises where you add to the solution as advance through the workshop—books such as Elixir in Action, or the mix introduction on the Elixir website, are both great examples of this.

So, the main thing would be losing the ability to create the force-skip command. Do you have any thoughts or ideas on how we could have both?

**Comment by Tuxified** [comment]
In the force solution situation, we could copy over the solution files?

[p.s.]
Sorry for the late reply, didn't see your response earlier

**Comment by gausby** [comment]
That's okay. I've been away from this project for a while, I need to look into the current features and such as it has been a really, really long time :)…I guess copying the solution into the folder would be great, as the student could have a peep at it.
